lbr_fri_ros2/src/interfaces/torque_command.cpp

```diff
@@ -36,6 +36,9 @@ void TorqueCommandInterface::buffered_command_to_fri(fri_command_t_ref command,
   }
   joint_position_filter_.compute(command_target_.joint_position, command_.joint_position);
 
+  command_.torque = command_target_.torque;
+  command_.joint_position = command_target_.joint_position;
```

Comment:
looks good overall, this line

```cpp
command_.joint_position = command_target_.joint_position;
```

is not necessary since `joint_position_filter_.compute` already does that

Reply:
I see you already did it with a pull request on our branch, so we should be fine now. 🙂